docs/source-app/workflows/byoc/index.rst

```diff
@@ -5,62 +5,270 @@ Run Apps on your own cloud (BYOC)
+   </video>
+
+----
+
+Method 2: Use Terraform to set up AWS (advanced)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This process is intended for advanced users only. If you have not used Terraform before, you might want to consider using the AWS console.
+
+#. `Download and install Terraform. <https://www.terraform.io/downloads>`_ If you haven’t used Terraform, `here is their tutorial <https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=terraform_io_download>`_.
+
+#. Install `AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html>`_.
+
+#. Using the AWS CLI log into your `AWS account <https://aws.amazon.com/free>`_ using `associated credentials <https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html>`_ that allow you to create resources. For example:
+
+    .. code:: bash
+
+        aws configure sso
+        aws sso login
+
+#. Clone the `terraform-aws-lightning-byoc <https://github.com/Lightning-AI/terraform-aws-lightning-byoc>`_ repo.
+
+#. Go to the quick-start directory inside the cloned repo.
+
+#. Run the following commands in the quick-start directory:
+
+    .. code:: bash
+
+        terraform init
+        export AWS_PROFILE=<your profile>
+        aws sts get-caller-identity
+        terraform apply
+
+#. Follow the directions in the CLI.
+
+#. Run the following to get the ARN and external ID:
+
+    .. code:: bash
+
+        terraform out -json
+
+If you are an IT person creating this role for someone else (researchers, data scientists, Lightning App creators), assign the role to their AWS account after creating the role.
+
+----
+
+Get the ARN and external ID
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once IT or you yourself have created the role, you need to get the ARN and external ID from the AWS console.
+
+Here's a short video showing you the process.
+
+.. raw:: html
+
+   <video id="background-video" autoplay loop muted controls poster="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/aws_signin.png" width="100%">
+      <source src="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/byoc_aws_get_arn_exid.mp4" type="video/mp4" width="100%">
+   </video>
+
+----
+
+*************************
+Step 2: Install Lightning
+*************************
+
+Lightning must be installed before you can create a Lightning BYOC cluster.
+
+**Who should install Lightning:**: Once your AWS account is ready, IT’s job is done. That means you (researchers, data scientists, Lightning App creators) are the one who will be doing this.
+
+----
+
+Install Lightning
+^^^^^^^^^^^^^^^^^
+
+From your local machine use the following command to install Lightning:
+
+.. code:: bash
+
+    python -m pip install lightning
+
+We have versioned pinned requirements at the time of the writing. Feel free to use the newer versions:
```

Comment:
No you have not; the 2 reqs aren't pinned

Reply:
@nmiculinic Please provide the pinning information

Reply:
@Felonious-Spellfire  

Please run the instruction, and then just run:
```
pip freeze
```

you'll see all dependencies and their version. Take that output and just pick the right versions for torchvision &  pytorch_lightning